ai2thor/tests/test_unity.py

```diff
@@ -33,6 +33,20 @@ def __init__(self,**kwargs):
     def prune_releases(self):
         pass
 
+# Defining const classes to lessen the possibility of a misspelled key
+# Review: This pattern is inconsistent with this file, but I think its a better direction
+# let me know if I should revert to match the rest of the file
```

Comment:
This pattern is inconsistent with this file, but I think its a better direction
let me know if I should revert to match the rest of the file